apps/consulting/pages/nebari-services.tsx

```diff
@@ -372,11 +372,8 @@ export const NebariServicesPage: FC<TContainerProps> = ({
     <section className="bg-violet">
       <h2 className="sr-only">Testimonials</h2>
       <div
-        className="p-4 mx-auto sm:px-[3.4rem] xl:px-[11.5rem] max-w-layout"
-        style={{
-          background:
-            'url(/nebari-services/quansight-logo-violet-grayscale.svg) right bottom / 325px no-repeat',
-        }}
+        // eslint-disable-next-line
+        className="py-8 px-4 mx-auto bg-[url(/nebari-services/quansight-logo-violet-grayscale.svg)] bg-right-bottom bg-no-repeat bg-[length:340px] sm:px-[3.4rem] md:py-32 md:bg-[length:460px] xl:px-[11.5rem] max-w-layout"
```

Comment:
This isn't *only* a simple transfer of `bg-[url(...)]` and its associated classnames into the Tailwind styles of the `<div>`:

- Padding was switched from `p-4` to `py-8` and `px-4` (twice the top/bottom padding)
- It looks like the old `/ 325px` became `bg-[length:340px]`?
- `md:py-32` added
- `md:bg-[length:460px]` added

Reply:
Yep, good eye, I made these changes to more closely match Irina's wire frame